mail_menager.py

```diff
@@ -0,0 +1,46 @@
+from db_menager import DB_Menager
+from user_menager import User_Menager
+
+class Mail_Menager(DB_Menager):
+    def __init__(self) -> None:
+        DB_Menager.__init__(self)
+        
+    def create_mail(self, user_id_from) -> None:
+        email = input("Enter email recipient: ")
+        user_id_to = User_Menager().get_user_id(email)
+        body = input("Your message: ")
+        
+        mycursor = self.connection.cursor()
+        mycursor.execute("INSERT INTO emails (user_id_from, user_id_to, body) VALUES (%s, %s, %s)", (user_id_from, user_id_to, body)) 
+        self.connection.commit()
+
+    def get_unread(self, user_id_to):
+        mycursor = self.connection.cursor()
+        mycursor.execute("SELECT text FROM emails WHERE user_id_to = %s AND is_read = 1", ([user_id_to]))
+        unread_mails = mycursor.fetchall()
+        return unread_mails
+
+    def print_outgoing(self, user_id_from):
```

Comment:
I think I shouldn't _PRINT_ all this stuff 
Or it's OK?
Or it's better to do like _RETURN_ value and in the _main.py_ to print it all ?

Reply:
and the same in other methods

Reply:
better that you return and main or another user interface function will print on a screen